Makefile

```diff
@@ -43,7 +43,7 @@ errcheck:
 
 .PHONY: staticcheck
 staticcheck:
-	go install ./vendor/honnef.co/go/staticcheck/cmd/staticcheck
+	go install ./vendor/honnef.co/go/tools/cmd/staticcheck
```

Comment:
not part of the review, but not sure overriding the system version is all that desirable. just running it from there must be better, but has nothing to do with updating the deps

Reply:
Agreed that this is unideal. Fixed in #33.